native/chat/chat-thread-list.react.js

```diff
@@ -181,6 +197,13 @@ class ChatThreadList extends React.PureComponent<Props, State> {
           searchResults.has(item.threadInfo.id),
         );
       }
+      if (this.props.emptyItem && chatItems.length === 0) {
```

Comment:
Let's get `emptyItem` from `propsAndState`. It's working now when you switch dark mode only because `ChatThreadBackground` never changes `emptyItem`. But calling `this.props` directly here means that if `ChatThreadBackground` changes `emptyItem`, the change won't get picked up by `ChatThreadList`. That's a bad way to build things - a component should never assume that a prop it's being passed won't change.

I tried to cover this in my last review:

> The reason it's happening is because we're checking `this.props.emptyItem` directly in `listDataSelector`, instead of selecting it from `propsAndState`